smdebug/tensorflow/utils.py

```diff
@@ -234,7 +229,9 @@ def get_chief_worker_from_tf_config(tf_config_json: dict):
 
 
 def is_mirrored_strategy(strat):
-    return isinstance(strat, (tf.distribute.MirroredStrategy, ContribMirroredStrategy))
+    return isinstance(
+        strat, (tf.distribute.MirroredStrategy, tf.compat.v1.distribute.MirroredStrategy)
```

Comment:
ditching 1.13 at all?

Reply:
yes, 1.13 was never supported in ZCC experience and now with introduction of 2.x support we intend to support 1.14+.